### DISCUSSION TITLE
Can windows activedirectories use this tool?

### DISCUSSION THREAD

**Discussion by FSerKevin** [ltb-project/self-service-password#679] [Q&A] [answered]
When I configure the LDAP connection, I always remind me that I cannot access the LDAP server. How can I solve this problem

Thanks！

**Comment by FSerKevin** [comment]


**Reply by coudot** [reply]
Tagging the developer of the product to get a faster answer is really not a good behavior

**Reply by FSerKevin** [reply]
Sorry, because I want to get your guidance quickly

**Reply by coudot** [reply]
If you need professional services, you can contact my company (@Worteks), see https://ltb-project.org/professionalservices.html

**Comment by coudot** [comment, marked as answer]
The tool is compatible with Active Directory.

See documentation: https://self-service-password.readthedocs.io/en/latest/config_ldap.html#active-directory

**Reply by FSerKevin** [reply]
Please tell me which directory the CA certificate is stored in and how to configure its configuration file. Can you give me some guidance

**Reply by coudot** [reply]
Follow the documentation about SSL configuration: https://self-service-password.readthedocs.io/en/latest/config_ldap.html#server-address
